trino/dbapi.py

```diff
@@ -302,6 +302,22 @@ def setinputsizes(self, sizes):
     def setoutputsize(self, size, column):
         raise trino.exceptions.NotSupportedError
 
+    def _remove_trailing_semicolon(self, statement: str) -> str:
+        """
+        Remove a trailing semicolon from sql statement if present
+
+        :param statement: sql statement
+        :return: modified sql statement
+        """
+        if ';' not in statement:
```

Comment:
What if a string in a statement will contain a semicolon? 
```sql
select c1 from t1 where c1 = ';value' 
```

Reply:
@hovaesco 
I literally just did what you asked in your previous review, please be more clear.

The previous logic was sound.
This logic is also sound. 

If there is no semicolon present in the query, the logic exits early.
Otherwise, the query is evaluated for a trailing semicolon - and removed. Only trailing semicolons will be removed with this logic, not semicolons within a string. So this won't matter.